tasks/main.yml

```diff
@@ -1,11 +1,11 @@
 ---
+- name: Common tasks
+  import_tasks: common.yml
+
```

Comment:
any reason why this is now ran in the beginning?

Reply:
Yes -- there is stuff that is defined in here -- things like the realm and other stuff in the ipa.conf that is used later on.
Easiest to run this first rather than repeat ourselves.